pywemo/ssdp.py

```diff
@@ -9,17 +9,47 @@
 
 import requests
 
-from .util import etree_to_dict, interface_addresses
+from .ouimeaux_device.api.long_press import VIRTUAL_DEVICE_UDN
+from .util import etree_to_dict, get_ip_address, interface_addresses
 
 DISCOVER_TIMEOUT = 5
 
+LOG = logging.getLogger(__name__)
+
 RESPONSE_REGEX = re.compile(r'\n(.*)\: (.*)\r')
 
 MIN_TIME_BETWEEN_SCANS = timedelta(seconds=59)
 
+MULTICAST_GROUP = "239.255.255.250"
+MULTICAST_PORT = 1900
+
 # Wemo specific urn:
 ST = "urn:Belkin:service:basicevent:1"
 
+SSDP_REPLY = f"""HTTP/1.1 200 OK
```

Comment:
I'm assuming this is fine, but I do see from the wireshark data I have that the SSDP data seems to all end with windows `\r\n` line endings...

Reply:
I changed it to use \r\n style instead, just in case. Good catch.